### PR TITLE
🍠 Squash migrations

### PR DESCRIPTION
**Pull request by rowanseymour**
```
run collect_sql to ensure nobody made schema changes and forgot to update that

delete all app migrations directories

./manage.py makemigrations locations
manually remove org field on boundaryalias and dependency to first orgs migration
./manage.py makemigrations orgs
./manage.py makemigrations contacts
manually remove channel field on contacturn and dependency to first contacts migration
./manage.py makemigrations channels
manually remove msg field on channellog and dependency to first msgs migration
./manage.py makemigrations schedules
./manage.py makemigrations msgs
manually add HStoreExtension() as first operation (from django.contrib.postgres.operations import HStoreExtension)
./manage.py makemigrations flows
manually remove campaign_event field on flowstart and dependency to first campaigns migration
./manage.py makemigrations campaigns
./manage.py makemigrations triggers
./manage.py makemigrations ivr
./manage.py makemigrations templates
./manage.py makemigrations archives
./manage.py makemigrations airtime
./manage.py makemigrations policies
./manage.py makemigrations api
./manage.py makemigrations apks
./manage.py makemigrations public

mkdir temba/sql/migrations
cp temba/sql/current_functions.sql temba/sql/migrations/initial_functions.sql
cp temba/sql/current_indexes.sql temba/sql/migrations/initial_indexes.sql
cp temba/sql/current_triggers.sql temba/sql/migrations/initial_triggers.sql

manually add other version of contact_toggle_system_group function because currently collect_sql doesn't understand overloaded functions

./manage.py makemigrations sql --empty -n initial

manually write migration which
 * has last migration of all others apps as a dependency (i.e. so it runs last) 
 * uses InstallSQL to install initial_functions.sql, initial_indexes.sql and initial_triggers.sql

manually rename all migrations so their numbering comes immediately after any existing migrations

NB probably easier to rename migrations as you generate them so the dependencies in subsequent migrations are set correctly for you
```

### CODE REVIEW

File: temba/sql/current_indexes.sql [hunk 1]
```diff
@@ -1,16 +1,6 @@
--- Generated by collect_sql on 2019-02-14 20:05 UTC
-
-CREATE INDEX api_webhookevent_created_on_pending_non_flow
-ON api_webhookevent(created_on)
-WHERE status = 'P' AND event != 'flow';
+-- Generated by collect_sql on 2019-07-01 22:49 UTC
 
-CREATE INDEX api_webhookevent_modified_on_errored_non_flow
-ON api_webhookevent(modified_on)
```
Comment (rowanseymour) [inline temba/sql/current_indexes.sql]: i'm assuming this index was automatically deleted when we dropped this column

**Comment by nicpottier** [comment]
Is that a squash emoji? If so, nicely done!

**Comment by nicpottier** [comment]
Maybe I'm reading these diffs wrong, but shouldn't the initials be renamed to whatever the current version is for those apps? (IE, not 0001 but 00054 or whatever?)

**Comment by rowanseymour** [comment]
It's WIP as per the name

**Comment by nicpottier** [comment]
Oh hah, funny I checked for WIP but always look at the start, doh!

**Comment by rowanseymour** [comment]
Maybe we need a label?

File: temba/ivr/tasks.py [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 from django_redis import get_redis_connection
 
+from django.conf import settings
```
Comment (rowanseymour) [inline temba/ivr/tasks.py]: @nicpottier are your pre-commit checks working ok ?

Comment (nicpottier) [reply temba/ivr/tasks.py]: I don't think they are working for me actually. (had to rerun black on one of mine)
But maybe I haven't reconfigured it the way I should have?

Comment (rowanseymour) [reply temba/ivr/tasks.py]: Try committing some junk!

**Comment by ericnewcomer** [comment]
I'm sad I missed this. But had to come back to +1 the emoji.

**Comment by rowanseymour** [comment]
Not gonna lie. Pretty proud of myself there.

**Pull request merged**